src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.tsx

```diff
@@ -259,6 +259,8 @@ function FloatingActionButtonAndPopover(
             showCreateMenu();
         }
     };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    const selfDMReportID = useMemo(() => ReportUtils.findSelfDMReportID(), [isLoading]);
```

Comment:
This useMemo cause the issue I mentioned.

Reply:
Did you test the PR with high traffic account? @nkdengineer

Reply:
I just tested with the account I used to test other issues. What is the high-traffic account?

Reply:
Ideally, you should test your PR with a high traffic account, it's account with ton of reports, check it out here:
https://github.com/Expensify/App/blob/68e69623a8bca0dce5f384fa351d5cbb2214aeb4/contributingGuides/CONTRIBUTING.md#high-traffic-accounts

Reply:
@hungvu193 I just tested it in high traffic account and it works well